Automattic/pocket-casts-ios
# Tracks: Adds analytics to the Notifications settings

| 📘 Project: #154 | 🛫 Depends on: #306 |
|:---:|:---:|

- `settings_notifications_shown`: When the notification settings is displayed to the user
- `settings_notifications_new_episodes_toggled`: When the user toggles the New Episodes option
- `settings_notifications_podcasts_changed`: When the use changes which podcasts should receive notifications
- `settings_notifications_app_badge_changed`: When the user changes the app badge option

## To test

1. Go to the profile tab
2. Tap on the cog
3. Tap on Notifications
4. ✅ `🔵 Tracked: settings_notifications_shown`
9. Go through each setting and change its value
12. Check console and verify each setting has a corresponding event in console and the value displayed is the value that you selected
13. If the option changed is a switch then the event should be suffixed `_toggle` if it's a value change it should be suffixed with `_changed`

## Checklist
